lib/rails_db/table.rb

```diff
@@ -60,6 +60,7 @@ def self.model_name
         klass = ActiveRecord::Base.descendants.detect { |c| c.table_name == table_name }
       end
 
+      ransack_methods(klass)
```

Comment:
1. please rename to add_ransack_methods
2. add a dependency in gemspec file to include ransack >= 4, this way it will require all developers to upgrade (OR to not make it so strict just add a check in the code if Ransack::VERSION >= 4.0 and then call this method)

Reply:
I like the` if Ransack::VERSION >= 4.0`, but got strange error` NameError (uninitialized constant Ransack::VERSION)`.
![Screenshot 2023-05-04 at 12 58 36](https://user-images.githubusercontent.com/82406343/236178305-b3778405-7f19-495a-a2ad-31d2556195bc.png)
So, I can get version in way like this 
`ransack_gem = Gem.loaded_specs['ransack']`
      `add_ransack_methods(klass) if ransack_gem.version >= Gem::Version.new('4.0.0')`. But not sure if it will be okay 👾👾👾

Reply:
I think this is good solution, as long specs are passed this is good for me

Reply:
@igorkasyanchuk fixed ( ˘▽˘)っ♨